sdk/maps/maps-search-rest/samples-dev/reverseGeocoding.ts

```diff
@@ -32,19 +32,27 @@ async function main() {
   // const client = MapsSearch(credential, mapsClientId);
 
   /** Make the request. */
-  const response = await client.path("/search/address/reverse/{format}", "json").get({
-    queryParameters: { query: [37.337, -121.89] },
+  const response = await client.path("/reverseGeocode").get({
+    queryParameters: { coordinates: [-121.89, 37.337] },
   });
 
   /** Handle error response. */
   if (isUnexpected(response)) {
     throw response.body.error;
   }
 
-  /** Log the response body. */
-  response.body.addresses.forEach((address) => {
-    console.log(address.address.freeformAddress);
-  });
+  if (!response.body.features || response.body.features.length === 0) {
+    console.log("No results found.");
+  } else {
+    /** Log the response body. */
+    response.body.features.forEach((feature) => {
```

Comment:
Instead of using `.forEach` would a `for .. of` loop work here as well?

Reply:
Yes, already added to the latest change, thanks for the advice